src/e2ir.d

```diff
@@ -5759,6 +5753,28 @@ elem *filelinefunction(IRState *irs, Loc *loc)
 }
 
 /******************************************************
+ * Construct elem to run when an array bounds check fails.
+ * Returns:
+ *      elem generated
+ */
+elem *buildArrayBoundsError(IRState *irs, ref Loc loc)
```

Comment:
Can `loc` be `const` since it's passed by `ref` and not modified, as far as I can see.